File: parantier-front/src/entities/personalBookmark/types/personalBookmark.ts
export interface PersonalBookmark {
  id: number;
  title: string;
  url: string;
  description?: string;
  category?: string;
  sortOrder: number;
  createdAt: string;
  updatedAt: string;
}

export interface CreatePersonalBookmarkRequest {
  title: string;
  url: string;
  description?: string;
  category?: string;
  sortOrder?: number;
}

export interface UpdatePersonalBookmarkRequest {
  title?: string;
  url?: string;
  description?: string;
  category?: string;
  sortOrder?: number;
}

File: parantier-api/src/main/java/com/mapo/palantier/personal/bookmark/PersonalBookmark.java
package com.mapo.palantier.personal.bookmark;

import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PersonalBookmark {

    private Long id;
    private Long userId;
    private String title;
    private String url;
    private String description;
    private String category;
    private Integer sortOrder;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private LocalDateTime deletedAt;
}
